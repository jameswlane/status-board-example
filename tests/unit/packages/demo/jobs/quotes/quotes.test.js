const fileToTest = require('../../../../../../packages/demo/jobs/quotes/quotes');

describe('Quotes', function () {
  test('onRun to match snapshot', function () {
    expect(fileToTest.onRun).toMatchSnapshot();
  });
});