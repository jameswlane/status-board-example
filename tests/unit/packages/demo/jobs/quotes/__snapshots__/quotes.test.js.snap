// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes onRun to match snapshot 1`] = `[Function]`;
